doc/admin/pool_operations.md

```diff
@@ -61,6 +61,19 @@ $ dmg pool destroy --pool=${puuid}
 $ dmg system list-pools
 ```
 
+This will return a table of pool UUIDs and the ranks of their pool service
+replicas. For example:
+
+```
+$ dmg system list-pools
+localhost:10001: connected
+Pool UUID					Svc Replicas
+---------					------------
+2a8ec3b2-729b-4617-bf51-77f37f764194	0,1
+a106d667-5c5d-4d6f-ac3a-89099196c41a	0
+85141a07-e3ba-42a6-81c2-3f18253c5e47	0
```

Comment:
This looks good, although there seems to be an alignment issue even when viewed in rich diff mode. Is there any other information we can display here? e.g. pool used/free

Reply:
Whoops. I had "fixed" it because it looked weird when I pasted it into my editor, the actual output is tabbed correctly in my terminal. Copying it directly this time.